test/services/filter_tagged_content_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'test_helper'
+
+describe FilterTaggedContent do
+  describe '#valid?' do
+    it 'is valid for an unknown document collection' do
+      document = Document.new(
+        content_store_document_type: 'document_collection',
+        base_path: '/a-new-document-collection'
+      )
+
+      assert(
+        FilterTaggedContent.new.valid?(document),
+        "It should display document collections we have no record of"
+      )
+    end
+
+    it 'is not valid for content tagged to an unknown document collection' do
+      document = Document.new(
+        'content_store_document_type' => 'guide',
+        'title' => 'Content in document collection',
+        'document_collections' => [{
+          'link' => '/a-new-document-collection'
+        }]
+      )
+
+      refute(
+        FilterTaggedContent.new.valid?(document),
+        "It should not show content items tagged to unknown document collections"
+      )
+    end
+
+    it 'is not valid for a document collection that should not be surfaced' do
+      document = Document.new(
+        content_store_document_type: 'document_collection',
+        base_path: '/government/collections/national-curriculum-assessments-information-for-parents'
```

Comment:
Yes, unfortunately they are. I can stub the bit that reads from the file and returned mocked results instead, what do you think?